sites/helpers/safe_get.py

```diff
@@ -0,0 +1,24 @@
+import time
```

Comment:
🔧 Let's combine `safe_get.py`, `scrape_error.py` and `validate.py` into a single module called `requests.py`! From what I've seen, Python folks tend to group functions, small classes & other objects that are thematically related to one another into a single module, and the one-function-per-file pattern isn't as widespread as it is in, say, JavaScript (see [here](https://www.reddit.com/r/learnpython/comments/5odyok/one_function_per_file/) and the comments in [here](https://stackoverflow.com/questions/41727874/proper-way-of-implementing-one-function-per-file-in-python)).

It's by no means a hard-and-fast rule, though, so I'm definitely happy to be disagreed with!

Reply:
👍